gpbft/gpbft.go

```diff
@@ -1321,27 +1323,31 @@ func (c *convergeState) FindMaxTicketProposal(table PowerTable) ConvergeValue {
 	// only or we go to a new round. Eventually there is a round where the max ticket is held by a
 	// correct participant, who will not double vote.
 	var maxValue ConvergeValue
-	var minTicket float64 = math.Inf(1)
+	var minTicket *big.Float
+
+	const precision = 257
```

Comment:
Based on the upstream example. I assume there's a reason they use the extra precision bit.